doc/develop/tools/vscode.rst

```diff
@@ -0,0 +1,76 @@
+.. _vscode_ide:
+
+VS Code
+#######
+
+`Visual Studio Code`_ is a popular cross-platform IDE that supports C projects and has a rich set of
+extensions.
+
+This guide describes the process of setting up VS Code for Zephyr's
+:zephyr:code-sample:`blinky` sample in VS Code.
+
+The instructions have been tested on Linux. In terms of the CLion workflow, the steps would be the
```

Comment:
clion?

Reply:
I based the document on the CLion guide, then my colleague @lucbeaufils tested the instructions from scratch.
Fixed the forgotten ref